### DISCUSSION TITLE
Windows Terminal Preview v1.16.3463.0 and v1.16.3464.0

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#14556] [Announcements]
Finally, a servicing update for our beloved Preview users! Sorry we left you on read for so long... but we think you'll like the changes in this update. Hope they're worth the wait!

Windows Terminal v1.16.346{3,4}.0 contain the following backports from https://github.com/microsoft/terminal/tree/main:

# Changes

* **Vintage transparency now works on Windows 10!** Turn off `useAcrylic` to use it! (#14481) 
* You can now configure whether you are warned before you close more than one tab (#14419) (thanks @leejy12!) (with wording improvements (#14473) thanks to @crazo7924!)
* Multiple changes to the new text rendering engine:
   * We will now attempt to better scale block and box drawing glyphs (#14099)
   * It will no longer crash with a heap overflow when presented with a malformed DBCS sequence (#14275)
   * We've adjusted how glyphs are scaled when they are _way_ outside their bounding boxes in poorly-behaved fonts (#14085)
* We've started collecting information on how often FTCS marks and themes are used (#14356)
* We've tweaked the layout of the color schemes page to address some issues on Windows 10 (#14223)
* Using the mouse wheel to turn off transparency will now disable the acrylic material effect (#14193) (thanks @JerBast!)

# Bug Fixes

* Windows Terminal once again works on Windows N (no media) SKUs
* We've fixed the following issues that could lead to a local denial of service or worse (with many thanks to @dgl for reporting):
   * `OSC 9;9` will now reject malformed paths and not further propagate them into `wsl --cd` (#14093) **REMOTE CODE EXECUTION** CVE-2022-44702
   * `OSC 8` URIs will be limited to 2MB in length, following iTerm2 (and only 1024 bytes will be displayed in the tooltip) (#14198)
* We will no longer leak `OpenConsole` processes when they're running a Visual Studio Developer Shell (#14160)
* `experimental.useBackgroundImageForWindow` has 50% fewer rendering bugs! (#14456)
* We will no longer clip the progress ring in tab when the tab title is too long (#14167) (thanks @JonathanMeier!)
* `START /MAX` should now more reliably maximize the Terminal window when Terminal is set as the default console host (#14222)
* `--pos` will now override the "center on launch" setting when specified on the command line (#14229) (thanks @ianjoneill!)
* Bracketed paste, forced titles, and the default cursor shape are no longer randomly initialized (#14345)
   * This fixes an issue where pasted text would occasionally be incorrectly mangled (as opposed to correctly mangled...)
* `RIS` will no longer utterly tank Terminal's ability to handle `C1` control characters (#13969) (thanks @j4james!)
* Console applications will now be correctly identified as the owners of their associated "pseudo-console" window (#14196)
* <kbd>Ctrl+C</kbd> will now skip any currently-playing `DECPS` sounds (thanks @dgl for reporting) (#14214)
* Applications should more reliably receive close signals when you close a tab, and we will no longer unceremoniously terminate their console sessions (#14282)
* You can now duplicate unfocused tabs (#13964) (thanks @JerBast!)
* The Open in Terminal shell extension should appear more reliably on the Desktop context menu (#14048) (#14211)
* `intenseTextStyle` is now included properly in the JSON schema (whoops) (#14210) (thanks @neersighted!)
* JSON Schema: `startingDirectory` is now correctly identified as supporting `null` (#14408)

## Accessibility & Usability

* <kbd>Alt+Space</kbd> will now open the system menu in the Settings UI and Command Palette (#14221)
* Text boxes in the settings UI have been made more discoverable by screen readers (#14178)
* There will no longer no longer not be redundant tooltips in the Settings UI (#14244)
* The WPF control now supports accessibility notifications (#14097)
* The Settings UI title/breadcrumb bar is now readable by screen readers (#14180)

## Reliability

* An application calling `system()` on its main thread will no longer deadlock itself and Terminal (when Terminal is set as default) (#14195)
* We've fixed one source of deadlocks in windowing management for console applications (such as hanging when they're exiting...) (#14463)
* Launching thousands of instances of `cmd.exe` back to back will no longer take down your console session (MSFT PR !8072712, 86928bb48d9963101af1f302d5939c7486cd822e)
* We've worked around a miscompilation in Visual Studio 2022 Update 4 that results in an out-of-bounds read (MSFT PR !8189936, 3c104440a897e551b867225d9fb8cfe4ff158dd8)
* wpf: it is no longer possible to call `WriteString` with a null terminal instance (#14515)
* wpf: It is now no longer possible to resize the terminal down to 0x0 (#14467)

# Code Hygiene

* `ConsoleProcessList` has been given some love and modernized to improve some of our ConPTY infrastructure (#14421)

## Dependency Updates

_brought to you by @AtariDreams_

* IntervalTree.h (#14148)
* jsoncpp 1.9.3 => 1.9.5 (#14122)
* libpopcnt => 2.5 (#14140)
* XAML => 2.7.3 (#14123)

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.16.3463.0'>Windows Terminal Preview v1.16.3463.0 and v1.16.3464.0</a>.</em>

**Comment by ofek** [comment]
> There will no longer no longer not be redundant tooltips in the Settings UI

😂 well played

**Comment by jessey-git** [comment]
Seems that `winget` wasn't updated. Again?

**Comment by rboxman** [comment]
Winget remains stuck on version 1.16.2641.0
